xain_fl/grpc/coordinator.py

```diff
@@ -245,11 +258,11 @@ class Coordinator:
     Args:
         num_rounds (:obj:`int`, optional): The number of rounds of the training
             session. Defaults to 10.
-        required_participants(:obj:`int`, optional): The minimum number of
-            participants required to perform a round. Defaults to 10.
-        aggregator: (:class:`~.Aggregator`, optional): The type of aggregation
-            to perform at the end of each round. Defaults to
-            :class:`~.FederatedAveragingAgg`.
+        minimum_participants_in_round (:obj:`int`, optional): The minimum number of
```

Comment:
just out of curiosity, did you also consider doing this the other way round, as in: have `minimum_connected_participants` configurable, and `minimum_participants_in_round` be determined. I think it's equivalent, but did you think about this / why did you choose this way?

Reply:
the way i thought about it is that from the user's point of view, they may prefer to set the minimum number of participant in a round, because that's what is going to affect the accuracy the most (ie: making sure there are enough participants in a given round, so that the aggregated model reflects the average of at least a few participants). with the assumption that this is what they care the most, then setting this value first and calculating the second saves them some mental algebra as opposed to the other case, where they set the number of connected participants and then calculate a fraction of it. 

but i do see your point, we can try it out and if we get the feedback that it doesn't quite make sense to users we can always refactor it.